virtcontainers/container.go

```diff
@@ -1325,16 +1329,37 @@ func (c *Container) updateMemoryResources(oldResources, newResources ContainerRe
 		addMemDevice := &memoryDevice{
 			sizeMB: int(memHotplugMB),
 		}
-		_, err = c.sandbox.hypervisor.hotplugAddDevice(addMemDevice, memoryDev)
+		data, err := c.sandbox.hypervisor.hotplugAddDevice(addMemDevice, memoryDev)
 		if err != nil {
 			return err
 		}
-		newResources.MemMB = newMemMB
+		memoryAdded, ok := data.(int)
+		if !ok {
+			return fmt.Errorf("Could not get the memory added, got %+v", data)
+		}
+		newResources.MemMB = oldMemMB + uint32(memoryAdded)
 		if err := c.sandbox.agent.onlineCPUMem(0, false); err != nil {
 			return err
 		}
 	}
 	// hot remove memory unsupported
+	if oldMemMB > newMemMB {
+		// Memory is not updated was not updaetd
```

Comment:
Typo, but also could you make this comment a little clearer?

Reply:
@jcvenegas please address this comment. I agree we need a clearer comment here.